Source/settings/items.ts

```diff
@@ -10,6 +10,9 @@ module FullScreenPokemon {
             "gameStarted": {
                 "valueDefault": false
             },
+            "collectionKeys": {
+                "valueDefault": []
+            },
```

Comment:
I'm having trouble deciding whether to have collectionsKeys as a property of StateHolder. While it will be a copy of whats in ItemsHolder, I think its better than it not being a property and have the logic in StateHolder.